string words
int32 integer
float32 double
---
int32 result
string status
